cmd/codegen/generator/client_set.go

```diff
@@ -79,7 +94,37 @@ func GenerateClient(generators []string, groupsWithVersions []string, inputPacka
 				if err != nil {
 					return err
 				}
+				// If we have a semver, copy the semver's pkg directory contents to the normal pkg and delete the semver directory
+				if semVer != "" {
```

Comment:
so what's going on here?

Reply:
Basically, I had to hack things a _bunch_ to get our codegen to properly result in `github.com/jenkins-x/jx/v2/pkg/...` etc.

Reply:
because client_set generator does not handle this properly?

Reply:
Yeah. I tried a lot of permutations/tweaks but couldn't ever get it quite right. Some of the generators did the right thing, some didn't.